### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by switzersc-usds**
* Add info on Community channels (google group, chats)
* Restructure main section on background and core team

### CODE REVIEW

**Review by esfoobar-usds — DISMISSED** [review]
🍏

File: README.md [hunk 1]
```diff
@@ -2,15 +2,38 @@
...
 
-The Justice40 team is taking a community-first and open source approach to the product development of this tool. We encourage contributions in the form of discussion on issues in this repo and pull requests of documentation and code. Please see the [Contributing](#Contributing) section for more information.
+## Background
+The Justice40 initiative and screening tool were announced in an [Executive Order](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/01/27/executive-order-on-tackling-the-climate-crisis-at-home-and-abroad/) in January 2021, with the goal to launch an MVP of the tool by July 27, 2021. This tool will include interactive maps and an initial draft scorecard which federal agencies can use to prioritize historically overburdened and underserved communities for benefits in their programs.
+
+## Core team
+The core Justice40 team building this tool is a small group of designers, developers, and product managers from the US Digital Service in partnership with the Council on Enviornmental Quallity (CEQ).
```
[the remark this answers is not in the document]
Comment (NatHillardUSDS) [reply README.md:11]: Enviornmental -> Environmental

[the remark this answers is not in the document]
Comment (NatHillardUSDS) [reply README.md:11]: Quallity -> Quality

File: README.md [hunk 2]
```diff
@@ -2,15 +2,38 @@
...
+
+## Community
+The Justice40 team is taking a community-first and open source approach to the product development of this tool. We believe government software should be made in the open and be built and licensed such that anyone can take the code, run it themselves without paying money to third parties or using proprietary software, and use it as they will. 
+
+We know that we can learn from a wide variety of communities, including those who will use or will be impacted by the tool, who are experts in data science or technology, or who have experience in climate, economic,or environmeental justice work. We are dedicateed to creating forums for continuous conversation and feedback to help shape the design and developmeent of the tool.
```
[the remark this answers is not in the document]
Comment (NatHillardUSDS) [reply README.md:16]: environmeental -> environmental

[the remark this answers is not in the document]
Comment (NatHillardUSDS) [reply README.md:16]: developmeent -> development

File: README.md [hunk 3]
```diff
@@ -2,15 +2,38 @@
...
+
+We know that we can learn from a wide variety of communities, including those who will use or will be impacted by the tool, who are experts in data science or technology, or who have experience in climate, economic,or environmeental justice work. We are dedicateed to creating forums for continuous conversation and feedback to help shape the design and developmeent of the tool. 
+
+We also recognize capacity building as a key part of involving a diverse open source community. We are doing our best to use accesible language, provide technical and process documents in multiple languages, and offer support to our community members of a wide variety of background and skillsets, directly or in the form of group chats and training. If you have ideas for how we can improve or add to our capacity building efforts and methods for welcoming folks into our community, please let use know in the [Google Group](https://groups.google.com/u/4/g/justice40-open-source) or email us at justice40open@usds.gov.
```
Comment (NatHillardUSDS) [inline README.md:18]: accesible -> accessible

File: README.md [hunk 4]
```diff
@@ -2,15 +2,38 @@
...
+
+Community members are welcome to share updates or propose topics for discussion in community chats. Please do so in the Google Group.
+
+### Google Group
+Our [Google Group](https://groups.google.com/u/4/g/justice40-open-source) is open to anyone to join and share their knowledge or experiences, as well as to ask questions of the core Justice40 team or the wider community. 
+
+The core team uses the group to post updates on the program and tech/data issues, and to share the agenda and call for community participation in the community chat.
+
+Curious about whether to ask a question here as a Github issue or in the Google Group? The general rule of thumb is that issues are for actionable topics related to the tool or data itself (e.g. questions about a specific data set in use, or suggestion for a new tool feature), and the Google Group is for more general topics or questions. If you can't decide, use the google group and we'll dicuss there before moving to Github if appropriate!
```
Comment (NatHillardUSDS) [inline README.md:30]: dicuss -> discuss

**Review by NatHillardUSDS — CHANGES_REQUESTED** [review]
Small nits / typos and adding a callout to the group

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (switzersc-usds) [reply README.md:11]: Adding a code owners file

**Review by NatHillardUSDS — APPROVED** [review]
LGTM!

**Pull request merged**